1.8/usage/service-discovery/haproxy-adminrouter.md

```diff
@@ -0,0 +1,107 @@
+
+1.  Create an HAProxy configuration for DC/OS. This example is for a DC/OS cluster on AWS. For more information on HAProxy configuration parameters, see the [documentation](https://cbonte.github.io/haproxy-dconv/configuration-1.6.html#3).
+
+    ```
+    global
+      daemon
+      log 127.0.0.1 local0
+      log 127.0.0.1 local1 notice
+      maxconn 20000
+      pidfile /var/run/haproxy.pid
+    defaults
+      log            global
+      option         dontlog-normal
+      mode		 http
+      retries             3
+      maxconn          20000
+      timeout connect  5000
+      timeout client  50000
+      timeout server  50000
+    
+    frontend http
+      # Bind on port 9090. HAProxy will listen on port 9090 on each 
+      # available network for new HTTP connections.
+      bind 0.0.0.0:9090
+      # Name of backend configuration for DC/OS.
+      default_backend dcos
+    
+      # Store request Host header temporarily in transaction scope
+      # so that its value is accessible during response processing.
+      # Note: RFC 7230 requires clients to send the Host header and
+      # specifies it to contain both, host and port information.
+      http-request set-var(txn.request_host_header) req.hdr(Host)
+    
+      # Overwrite Host header to 'dcoshost'. This makes the Location
+      # header in DC/OS Admin Router upstream responses contain a
+      # predictable hostname (nginx uses this header value when
+      # constructing absolute redirect URLs). That value is used
+      # in the response Location header rewrite logic (see regular
+      # expression-based rewrite in the backend section below).
+      http-request set-header Host dcoshost
+    
+    backend dcos
+      # Option 1: use TLS-encrypted communication with DC/OS Admin Router and
+      # perform server certificate verification (including hostname verification).
+      # If you are using the community-supported version of DC/OS, you must 
+      # configure Admin Router with a custom TLS server certificate, see 
+      # https://dcos.io/docs/1.8/administration/securing-your-cluster/. This step 
+      # is not required for Enterprise DC/OS.
+      #
+      # Explanation for the parameters in the following `server` definition line:
+      # 
+      # 1.2.3.4:443
+      #
+      #   IP address and port that HAProxy uses to connect to DC/OS Admin
+      #   Router. This needs to be adjusted to your setup.
+      #   
+      #
+      # ssl verify required
+      #
+      #   Instruct HAProxy to use TLS, and to error out if server certificate
+      #   verification fails.
+      #
+      # ca-file dcos-ca.crt
+      #
+      #   The local file `dcos-ca.crt` is expected to contain the CA certificate
+      #   that Admin Router's certificate will be verified against. It must be
+      #   retrieved out-of-band (on Mesosphere Enterprise DC/OS this can be
+      #   obtained via https://dcoshost/ca/dcos-ca.crt)
```

Comment:
I should have responded more exhaustively to your original comment.

> not sure if it is the same for open.

This is described a few lines above:

```
+      # If you are using the community-supported version of DC/OS, you must 
+      # configure Admin Router with a custom TLS server certificate, see 
+      # https://dcos.io/docs/1.8/administration/securing-your-cluster/.
```

Also, I meant to say "`on Mesosphere Enterprise DC/OS this can be obtained via https://dcoshost/ca/dcos-ca.crt` does not curently apply to Open DC/OS" (as hopefully is apparent from the sentence itself).

> It might be a good thing to mention somewhere that this section is applicable to Enterprise only. 

Hope it's clear now that this is not true. The configuration shown is valid for both, literally. What's different is _how_ to obtain `dcos-ca.crt`. This is covered for both scenarios 
* via linking to other content in case of Open DC/OS and
* via a short description in case of Enterprise.

Ok?